cadc-vos-server/src/main/java/org/opencadc/vospace/server/transfers/InternalTransferAction.java

```diff
@@ -138,15 +138,16 @@ public void doAction() throws Exception {
             log.debug("checking move permissions: " + srcURI + " -> " + destURI);
 
             // resolve the links to containers in the path so we get the actual node
-            PathResolver res = new PathResolver(nodePersistence, authorizer, true);
-            Node srcNode = res.getNode(srcURI.getPath());
+            PathResolver res = new PathResolver(nodePersistence, authorizer);
+            Node srcNode = res.getNode(srcURI.getPath(), true);
```

Comment:
I'm not sure resolveLeafLink==true is right here. It means you don't move a LinkNode and end up moving the target of the link node, which also breaks the link node.... it was probably always a bug that was hiding due to the ambiguous meaning of "resolve"

at least the TODO comment below is no longer a concern and can be deleted :-)